src/data/Profile.json

```
@@ -168,6 +168,18 @@
       "LinkedIn": "https://www.linkedin.com/in/kshitiz-rana-264457226/"
     }
   },
+  {
+    "name": "Rafiya Rahman",
+    "location": "kolkata, India",
+    "bio": "I am currently learning to contribute in open source.I am a first year student, eager and open to learn and grow",
+    "avatar": "https://github.com/rafiya2003.png",
+    "portfolio": "https://github.com/rafiya2003",
+    "skills": ["Python", "HTML", "Java" "..."],
```

```suggestion
    "skills": ["Python", "HTML", "Java"],
```
remove `"..."` from skills array.

---

@rafiya2003  remove `"..."` from skills array